documentation/book/ref-tracing-environment-variables.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// assembly-setting-up-tracing-kafka-clients.adoc
+
+[id='ref-tracing-environment-variables-{context}']
+= Tracing environment variables
+
+Use these environment variables to configure a Jaeger tracer for Kafka client applications.
```

Comment:
We should make clear that these are not our variables but Jaegers and they are subject to change by Jaeger and ideally add a link to their docs for the description in case they change.